package.json

```diff
@@ -54,8 +54,9 @@
     "eslint-plugin-react": "4.2.3",
     "express": "4.13.4",
     "jsdom": "8.3.0",
-    "react-addons-test-utils": "0.14.8",
+    "react": "0.14.8",
```

Comment:
some devDependencies (enzyme, etc) still rely on `react` and `react-dom`, so we keep them around—webpack's resolver *will still them as externals* when imported in development mode.